cc-env.go

```diff
@@ -330,16 +337,21 @@ func getEnvInfo(configFile, logfilePath string, config oci.RuntimeConfig) (env E
 		Resolved: resolvedHypervisor.KernelPath,
 	}
 
+	kernelParams := KernelParameters{
+		Parameters: strings.Join(vc.SerializeParams(config.HypervisorConfig.KernelParams, "="), " "),
+	}
+
 	env = EnvInfo{
-		Meta:       meta,
-		Runtime:    ccRuntime,
-		Hypervisor: hypervisor,
-		Image:      image,
-		Kernel:     kernel,
-		Proxy:      ccProxy,
-		Shim:       ccShim,
-		Agent:      ccAgent,
-		Host:       ccHost,
+		Meta:         meta,
+		Runtime:      ccRuntime,
+		Hypervisor:   hypervisor,
+		Image:        image,
+		Kernel:       kernel,
+		KernelParams: kernelParams,
```

Comment:
Running `cc-runtime cc-env` gives the following:

```
[Kernel]
  Path = "/usr/share/clear-containers/vmlinux.container"
  Resolved = "/usr/share/clear-containers/vmlinux-4.9.35-62.container"

[KernelParams]
  Parameters = ""
```

But what we really want is:

```
[Kernel]
  [Kernel.Location]
    Path = "/usr/share/clear-containers/vmlinux.container"
    Resolved = "/usr/share/clear-containers/vmlinux-4.9.35-62.container"
  Parameters = ""
```

That way, all the kernel details are kept under `Kernel`. This is also in line with what we already have for `Hypervisor`:

```
[Hypervisor]
  MachineType = "pc-lite"
  [Hypervisor.Location]
    Path = "/usr/bin/qemu-lite-system-x86_64"
    Resolved = "/usr/bin/qemu-lite-system-x86_64"
```